### PR TITLE
Update schema for profile and membership

### PR DESCRIPTION
**Pull request by swilczynski**
## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Profile.yaml [hunk 1]
```diff
@@ -29,11 +29,21 @@ properties:
     readOnly: true
     type: string
     nullable: true
+  permissions:
```
Comment (ashkarpetin) [inline openapi/components/schemas/Profile.yaml:32]: I think this should be readOnly

File: openapi/components/schemas/Profile.yaml [hunk 2]
```diff
@@ -60,6 +76,14 @@ properties:
     type: string
     format: url
     deprecated: true
+  status:
+    description: Status of the user's account.
```
Comment (nanobash) [inline openapi/components/schemas/Profile.yaml:80]: I think here `'s` is not needed, or mistaken @noon-dawg?

```suggestion
    description: Status of the user account.
```

File: openapi/components/schemas/Profile.yaml (hunk 1 above)
Comment (swilczynski) [reply openapi/components/schemas/Profile.yaml:32]: Thanks, I'll fix it

File: openapi/components/schemas/Membership.yaml [hunk 3]
```diff
@@ -56,5 +61,14 @@ properties:
       For example, the user may be an organization admin.
     items:
       type: string
+  acl:
+    $ref: ./Acl.yaml
```
Comment (ashkarpetin) [inline openapi/components/schemas/Membership.yaml:65]:
```suggestion
    readOnly: true
    allOf:
      - $ref: ./Acl.yaml
```

**Review by ashkarpetin — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Pull request merged**